v2/pkg/catalog/loader/loader.go

```diff
@@ -313,8 +314,11 @@ func (store *Store) LoadTemplatesWithTags(templatesList, tags []string) []*templ
 				}
 				gologger.Warning().Msgf("Could not parse template %s: %s\n", templatePath, err)
 			} else if parsed != nil {
+
 				if len(parsed.RequestsHeadless) > 0 && !store.config.ExecutorOptions.Options.Headless {
 					gologger.Warning().Msgf("Headless flag is required for headless template %s\n", templatePath)
+				} else if len(parsed.RequestsCode) > 0 && !parsed.Verified {
+					gologger.Warning().Msgf("The template is not verified: '%s'\n", templatePath)
```

Comment:
@Mzack9999 , I think we should adopt a different strategy here . this will add lot of noise , users run custom templates on regular basis . so I was thinking just summarizing would be enough and we can print list of template id's if `-v` flag is used . ref: https://github.com/projectdiscovery/nuclei/pull/3635/commits/ed814332cb6a402973f407935d0d5c87da5d8592

```
Found xxx unverified templates . Use With Caution
```
or something similar to this ^ .

cc: @ehsandeep

Reply:
I agree it will be very verbose. What do you think @ehsandeep ?